src/panels/utilities/ClusterSpecCreationBuilder.ts

```diff
@@ -27,7 +30,41 @@ export class ClusterDeploymentBuilder {
         },
     };
 
-    public buildCommonParameters(clusterSpec: ClusterSpec): ClusterDeploymentBuilder {
+    public buildCommonParameters(clusterSpec: ClusterSpec, preset: string): ClusterDeploymentBuilder {
+        return preset === "automatic"
```

Comment:
💡 Observation only please: what will happen if case sensitivity comes into play, in case we want to handle this also in case this is also candidate of `const` or `enum`